### DISCUSSION TITLE
mkdocstrings is now included in cookiecutter template

### DISCUSSION THREAD

**Discussion by zillionare** [mkdocstrings/mkdocstrings#290] [Show and tell]
I'm fan of mkdocstrings (and mkdocs). I've made a cookiecutter template, and included mkdocstrings. Once you've created a skeleton project from [this template](https://github.com/zillionare/cookiecutter-pypackage), mkdocstrings is all set.

**Comment by pawamoy** [comment]
Since we're sharing templates using mkdocstrings, here are mine :stuck_out_tongue: 

- https://github.com/pawamoy/copier-pdm
- https://github.com/pawamoy/copier-poetry
